src/ResourceManagement/RecoveryServices.Backup/RecoveryServicesBackupManagement/Generated/ProtectionPolicyOperations.cs

```diff
@@ -195,153 +195,160 @@ public RecoveryServicesBackupManagementClient Client
                     {
                         JObject propertiesValue = new JObject();
                         itemValue["properties"] = propertiesValue;
+                        if (request.Item.Properties is ProtectionPolicy)
+                        {
+                            propertiesValue["backupManagementType"] = "ProtectionPolicy";
+                            ProtectionPolicy derived = ((ProtectionPolicy)request.Item.Properties);
+                            
+                            propertiesValue["protectedItemsCount"] = derived.ProtectedItemsCount;
+                        }
                         if (request.Item.Properties is AzureIaaSVMProtectionPolicy)
                         {
                             propertiesValue["backupManagementType"] = "AzureIaasVM";
-                            AzureIaaSVMProtectionPolicy derived = ((AzureIaaSVMProtectionPolicy)request.Item.Properties);
+                            AzureIaaSVMProtectionPolicy derived2 = ((AzureIaaSVMProtectionPolicy)request.Item.Properties);
                             
-                            if (derived.SchedulePolicy != null)
+                            if (derived2.SchedulePolicy != null)
                             {
                                 JObject schedulePolicyValue = new JObject();
                                 propertiesValue["schedulePolicy"] = schedulePolicyValue;
-                                if (derived.SchedulePolicy is SimpleSchedulePolicy)
+                                if (derived2.SchedulePolicy is SimpleSchedulePolicy)
```

Comment:
Can we have better have specifying the class of object. This is going to be difficult to track later. object of AzureIAASVMProtectionPolicy can be iaasPol, object of SimpleSchedulePolicy can be simPol, something like that will be better